dspace-spring-rest/src/main/java/org/dspace/app/rest/model/DSpaceObjectRest.java

```diff
@@ -20,13 +20,16 @@
  *
  */
 public abstract class DSpaceObjectRest extends BaseObjectRest<String> {
+	public static final String PRJ_DEFAULT = "";
+	public static final String PRJ_CONTEXT = "context";
```

Comment:
Wouldn't it be better to not hard-code the possible projection values in the code, but to define them in Spring? That would allow an institution to add new projection types without having to modify the DSpace source code.

We could define something like this in Spring XML:
```
<util:map id="restProjections">
        <entry key="community" value-ref="restCommunityProjections" />
        <entry key="item" value-ref="restItemProjections"/>
</util:map>

<util:map id="restCommunityProjections">
        <entry key="context" value-ref="CommunityContextProjectionApplier"/>
        <entry key="" value-ref="CommunityDefaultProjectionApplier"/>
</util:map>
```

Or do something clever with the Spring ID names (`<DSpaceObject Type> + <Projection Name> + 'ProjectionApplier'`, e.g. "CommunityContextProjectionApplier").